pkg/services/services_test.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"testing"
+
+	config "github.com/openshift/api/config/v1"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetHostnameCmd(t *testing.T) {
+	tests := []struct {
+		name         string
+		platformType config.PlatformType
+		expected     string
+	}{
+		// TODO: Add test cases.
```

Comment:
This TODO isn't helpful as it doesn't describe what test cases are missing.
Additionally if you know what test cases are missing that are required, please just add them

Reply:
Missed removing this line. Good catch.